### PR TITLE
(fix) converted editing billable service form from overlay to workspace

### PR DESCRIPTION
**Pull request by FelixKiprotich350**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Converted editing billable service form from overlay to workspace

## Screenshots
[screen-capture (6).webm](https://github.com/user-attachments/assets/5a944ff2-395d-471c-8d60-11a685736763)

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by FelixKiprotich350** [comment]
hey @donaldkibet  and @ojwanganto check this workspace

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss [hunk 1]
```diff
@@ -0,0 +1,168 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@use '@carbon/colors';
+@use '@carbon/layout';
+@import '~@openmrs/esm-styleguide/src/vars';
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss:5]:
```suggestion

```
We don't need this import. All styles can be derived from respective carbon imports

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss [hunk 2]
```diff
@@ -0,0 +1,168 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss:2]:
```suggestion
@use '@carbon/layout';
@use '@carbon/type';
```

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  padding: 0px;
+}
+
+.subTitle {
+  font-weight: 600;
+  font-size: 14px;
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss:17]: we should the typography classes provided by carbon e.g

```scss
@include type.type-style('heading-01');
```
See https://carbondesignsystem.com/elements/typography/type-sets

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  border-right: 1px solid colors.$cool-gray-40;
+}
+
+.paymentButtons {
+  margin: layout.$layout-01 0;
+}
+
+.paymentMethodContainer {
+  display: grid;
+  grid-template-columns: 40% 30% 20%;
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss:47]: Are better of using carbon grid hee?

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+.spinner {
+  &:global(.cds--inline-loading) {
+    min-height: 1rem;
+  }
+}
+
+.loader {
+  margin-top: 1rem;
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.scss:135]: use carbon token for layout

**Review by donaldkibet — CHANGES_REQUESTED** [review]

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.workspace.tsx [hunk 6]
```diff
@@ -0,0 +1,400 @@
...
+            name="shortName"
+            render={({ field }) => (
+              <Layer>
+                <TextInput
+                  id="serviceShortName"
+                  {...field}
+                  type="text"
+                  labelText={<span className={styles.subTitle}>{t('serviceShortName', 'Short Name')}</span>}
+                  size="md"
+                  placeholder="Enter service short name"
```
Comment (makombe) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.workspace.tsx:210]: We should also translate the placeholder here

File: packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.workspace.tsx [hunk 7]
```diff
@@ -0,0 +1,400 @@
...
+                  invaliShort
+                  Name
+                  d={!!errors.shortName}
+                />
+              </Layer>
+            )}
+          />
+        </div>
+        <div className={styles.itemSection}>
+          <FormLabel className={styles.subTitle}>Associated Concept</FormLabel>
```
Comment (makombe) [inline packages/esm-billing-app/src/billable-services/create-edit/workspaces/edit-billable-service.workspace.tsx:221]: Lets translate this as well

**Review by makombe — COMMENTED** [review]
Thanks @FelixKiprotich350 generally looks good.

**Comment by FelixKiprotich350** [comment]
@makombe done

**Comment by FelixKiprotich350** [comment]
@makombe i have fixed the issue discussed earlier

**Review by makombe — APPROVED** [review]
Thanks @FelixKiprotich350

**Comment by ojwanganto** [comment]
Thanks, @FelixKiprotich350 @donaldkibet @makombe

**Pull request merged**